docusaurus/docs/reference/tunnelers/80-kubernetes/index.mdx

```diff
@@ -1,5 +1,6 @@
 ---
-title: Workload Tunneling
+title: Tunneling Kubernetes Workloads
```

Comment:
The previous title works better for me. It's kinda redundant to see Kubernetes in the title when i just clicked the Kubernetes sidebar thingy. Also it makes the title wrap even when the windows is modestly sized. I'd like to revert this